api/services/SandboxHelper.js

```diff
@@ -0,0 +1,76 @@
+const moment = require('moment');
+const {
+  Op, fn, col, where: whereClause,
+} = require('sequelize');
+const PromisePool = require('@supercharge/promise-pool');
+const Mailer = require('./mailer');
+const {
+  User, Organization, Site, Role,
+} = require('../models');
+const SiteDestroyer = require('./SiteDestroyer');
+const { sandboxDays } = require('../../config').app;
+
+const notifyOrganizations = async (cleaningDate) => {
+  const dateStr = moment(cleaningDate).format('YYYY-MM-DD');
```

Comment:
as suggested, the function takes the date parameter which is the date the sandbox organization is scheduled to be clean.  the sandboxDaysNotice is taken into consideration by the function caller. https://github.com/18F/federalist/blob/96c385bbbc58ef6ef74ed85db149cdca90cfde08/api/workers/jobProcessors/sandboxNotifications.js#L7

Reply:
aaaaaahhhhh. So when I saw this function, I thought it would mean

```
send the notifications that should be sent on `cleaningDate`
```
not

```
send the notifications that should be sent for CLEANING DATE `cleaningDate`
```

Obvi, the variable name makes more sense for the latter meaning so maybe I am just an idiot :)

A function that does the former seems to be easier for my brain to understand, but this is probably just personal preference so no worries.